src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
 public @interface Inject {
```

Comment:
The `@Inject` annotation is missing the `@Target` annotation. To follow best practices, you should specify where this annotation can be applied using `@Target`. Consider adding it to enhance the clarity and correctness of your annotation.